src/codecoverage/bot/code_coverage_bot/secrets.py

```diff
@@ -16,6 +16,7 @@ class Secrets(dict):
     GECKO_DEV_PWD = 'GECKO_DEV_PWD'
     HG_GIT_MAPPER = 'HG_GIT_MAPPER'
     EMAIL_ADDRESSES = 'EMAIL_ADDRESSES'
+    REPO_MAPPER_EMAIL_ADDRESSES = 'REPO_MAPPER_EMAIL_ADDRESSES'
```

Comment:
Add a default of an empty array for this, you can add a default by adding an `existing` parameter to the call to `get_secrets` a few lines after this

Reply:
U mean `existing={REPO_MAPPER_EMAIL_ADDRESSES:[]}`

Reply:
Yes!

Reply:
Of course, REPO_MAPPER_EMAIL_ADDRESSES has to be a string

Reply:
Also, is there any slack channel, like in general how do I communicate?